externs/shaka/player.js

```diff
@@ -293,6 +294,9 @@ shaka.extern.BufferedInfo;
  *   The count of the audio track channels.
  * @property {?number} audioSamplingRate
  *   Specifies the maximum sampling rate of the content.
+ * @property {boolean} spatialAudio
+ *   True indicates that this in the spatial audio for the content.
```

Comment:
nit: something's going with this comment - either a word is missing or mistyped, or something like that

Reply:
Changed!